.github/workflows/ci.yml

```diff
@@ -223,9 +234,23 @@ jobs:
     secrets: inherit
     with:
      os: linux
+     runner: ubuntu-latest
      app: dl
      cluster: ${{ needs.deploy-cluster.outputs.clusterName }}
 
+  dart-tests-linux20:
+    name: Linux 20 Dart Tests
```

Comment:
also lets rename the names here. Linux Dart Tests (ubuntu 20)

Reply:
Done